plugins/wasm-rust/DockerfileBuilder

```diff
@@ -0,0 +1,38 @@
+
```

Comment:
why do we need another Dockerfile to build?

Reply:
I referred to the builder used for packaging and publishing in the go-lang https://github.com/alibaba/higress/blob/main/plugins/wasm-go/DockerfileBuilder

Reply:
Dockerfile is enough, no need additional files

Reply:
Both Dockerfile and DockerfileBuilder are for building an artifact named plugin.wasm, but the oras is a client which aims to publish artifact to registry. It means no matter which language is, the only different is how to build an artifact but using same way to publish artifact. 

In a word, I think the oras should be a independent image, but not mixed in builders. 

cc @johnlanni

Reply:
builder mainly facilitates developers by eliminating the need to download corresponding versions of oras, as well as Rust build tools, etc

Reply:
For developers, devcontainer might be the best solution and I still think it should keep simple for OCI building

Reply:
Well, you're right, developers just need the Dockerfile, but it seems that the DockerfileBuilder used for building images allows the workflow for publishing wasm plugins to remain consistent with Go. From this perspective, it still makes sense